packages/freezed/test/builder_test.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:convert';
+
+import 'package:build/build.dart';
+import 'package:build_test/build_test.dart';
+import 'package:freezed/builder.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('defaultFormatOutput formats code correctly', () {
```

Comment:
That's a bit complex and likely to break.
I'd just read a file from the `integration` folder and check if it contains the comment or not

Reply:
> I'd just read a file from the `integration` folder and check if it contains the comment or not

Do you have a preferred place where the test should be put?